### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by YuriKravetc**

### CODE REVIEW

File: docs/Cluster-VictoriaMetrics.md [hunk 1]
```diff
@@ -545,7 +545,7 @@ Below is the output for `/path/to/vminsert -help`:
   -influxDBLabel string
      Default label for the DB name sent over '?db={db_name}' query parameter (default "db")
   -influxListenAddr string
-     TCP and UDP address to listen for InfluxDB line protocol data. Usually :8189 must be set. Doesn't work if empty. This flag isn't needed when ingesting data over HTTP - just send it to http://<victoriametrics>:8428/write
+     TCP and UDP address to listen for InfluxDB line protocol data. Usually :8089 must be set. Doesn't work if empty. This flag isn't needed when ingesting data over HTTP - just send it to http://<victoriametrics>:8428/write
```
Comment (hagen1778) [inline docs/Cluster-VictoriaMetrics.md:548]: Why you think `8089` should be recommended instead `8189`?

Comment (YuriKravetc) [reply docs/Cluster-VictoriaMetrics.md:548]: Seems like the default port that runs the UDP service **usually** is 8089. https://docs.influxdata.com/influxdb/v1.8/administration/ports/

Comment (denisgolius) [reply docs/Cluster-VictoriaMetrics.md:548]: Also 8089 used in our docker examples https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/docker-compose.yml#L25

**Review by hagen1778 — CHANGES_REQUESTED** [review]
@YuriKravetc please add these changes to the origin docs (in root's README and vmagent's README) and run `make docs-sync` command.

**Comment by YuriKravetc** [comment]
> @YuriKravetc please add these changes to the origin docs (in root's README and vmagent's README) and run `make docs-sync` command.

This is done and docs are in sync. Thanks for pointing it out.

**Review by f41gh7 — APPROVED** [review]
LGTM

File: docs/README.md [hunk 2]
```diff
@@ -1682,7 +1682,7 @@ Pass `-help` to VictoriaMetrics in order to see the list of supported command-li
   -influxDBLabel string
      Default label for the DB name sent over '?db={db_name}' query parameter (default "db")
   -influxListenAddr string
-     TCP and UDP address to listen for InfluxDB line protocol data. Usually :8189 must be set. Doesn't work if empty. This flag isn't needed when ingesting data over HTTP - just send it to http://<victoriametrics>:8428/write
+     TCP and UDP address to listen for InfluxDB line protocol data. Usually :8089 must be set. Doesn't work if empty. This flag isn't needed when ingesting data over HTTP - just send it to http://<victoriametrics>:8428/write
```
Comment (hagen1778) [inline docs/README.md:1685]: The change need to be applied to https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/README.md first. And then synced to /docs folder via `make docs-sync`

**Review by hagen1778 — CHANGES_REQUESTED** [review]

**Review by hagen1778 — APPROVED** [review]

**Pull request merged**